careerfairsystems/arkad-app-2018
# Design about screen

## Change description

Designs the About screen.

## How to verify

Go to the About screen tab and:
- Verify that information about ARKAD and opening hours is shown
- Verify that pressing the ARKAD team button takes you to a new view with information about the team

## Issues fixed

This fixes #35.
